steps/create-apple-s-documentation/1.0.0/step.yml

```diff
@@ -0,0 +1,132 @@
+title: Create Apple's documentation
+summary: |
+  This step helps to create a HTML representation of the Framework documentation, using Jazzy as engine
+description: |
+  This step helps to create a HTML representation of the Framework documentation, using Jazzy as engine.
+  Objective-C and Swift are e supported, but there is no declared supported for mixed language projects.
+  THis was created with the goal to export as a reach HTML documents for Frameworks.
+website: https://github.com/FutureWorkshops/bitrise-step-create-apple-s-documentation
```

Comment:
The step ID `create-apple-s-documentation` is valid, still `create-apple-documentation` would be easier to read.